pandevice/base.py

```diff
@@ -2934,27 +3001,37 @@ def set_vsys(self, vsys_id, refresh=False, update=False, running_config=False):
             update (bool): Apply the changes to the device (Default: False)
             running_config (bool): If refresh is True, refresh from the running
                 configuration (Default: False)
+            return_boolean (bool): Return a boolean saying if an update
+                is/was needed instead of a Vsys object.
```

Comment:
Expand on this to documentation to explain what a True and False value represents and how that relates to if it was updated or not.  Also consider a more descriptive parameter name.